ui/src/css/storyTelling.css

```diff
@@ -48,7 +48,21 @@ fieldset ul {
 
 .sjrk-st-page-content-container {
     background-color: #fff;
-    padding: 2rem;
+    padding: 0 2rem 2rem 2rem;
+}
+
+/* These sections are shown or hidden based on the theme's styling rules */
```

Comment:
Could you clarify this comment, regarding which themes are sharing this css flle and which theme will override these rules etc. Does "These sections" mean starting from here till the end of the file?